enforce-signer.tf

```diff
@@ -18,3 +18,9 @@ resource "google_project_iam_member" "signer_cert_requester" {
   role    = "roles/privateca.certificateRequester"
   member  = "serviceAccount:${google_service_account.chainguard_signer.email}"
 }
+
+resource "google_project_iam_member" "signer_encrypt_decrypter" {
```

Comment:
This grants access to all keys in the project - this might be too broad for some users. Fine for prototyping, but we probably will want to look into restricting this down through a var + a more specific resource like https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_kms_key_ring_iam

Reply:
I felt it is fine for now. Otherwise we could repeat what we're doing for the oidc-keyring here.

Reply:
I added a TODO comment about it.